### ISSUE TITLE
Milestone image update interval

### ISSUE THREAD

**Issue by lkeegan** [ssciwr/mondey#62]
Currently when there are multiple images for a [milestone](https://ssciwr.github.io/mondey-frontend-prototype/milestone), they are cycled every 5 seconds, and they cycle indefinitely.

Questions:

- should the interval between images be shorter/longer?
- should they cycle indefinitely, or only cycle once then stop updating?

**Comment by MaHaWo** [comment]
I was wondering if the [carousel component](https://flowbite-svelte.com/docs/components/carousel#Carousel_with_links) could be useful here?

**Comment by lkeegan** [comment]
Yes - I did start with this component, but getting it to resize nicely in the vertical&horizontal layouts ended up being more difficult than just directly implementing what I wanted - but we should certainly revisit this if we end up adding more carousel features
